### PR TITLE
Add inspect tests

### LINKED ISSUE #2121: Add basic `singularity inspect` test
Add basic `singularity inspect` test

### PR DESCRIPTION
**Pull request by jscook2345**
**Description of the Pull Request (PR):**

Add a basic `singularity inspect` test

**This fixes or addresses the following GitHub issues:**

- Fixes #2121 

**Checkoff for all PRs:**

- [X] I have read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [user](https://www.github.com/sylabs/singularity-userdocs) and [administrator](https://www.github.com/sylabs/singularity-admindocs) documentation bases.
- [X] I have tested this PR locally with a `make testall`
- [X] This PR is against the project's master branch
- [ ] I have added myself as a contributor to the [contributors's file](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)
- [X] This PR is ready for review and/or merge

Attn: @singularity-maintainers

### CODE REVIEW

File: src/cmd/singularity/inspect_test.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	cmd := exec.Command(cmdPath, args...)
+
+	return cmd.CombinedOutput()
+}
+
+func TestInspect(t *testing.T) {
+	var definition = "../../../examples/busybox/Singularity"
+	var image = "Default.sif"
+
+	if b, err := buildImage(definition, image); err != nil {
```
Comment (ArangoGutierrez) [inline src/cmd/singularity/inspect_test.go]: I think we could add a labels section to that busybox definition
```
%labels

    Maintainer sylabs

    Version v3.0.0-beta
```

File: src/cmd/singularity/inspect_test.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/sylabs/singularity/src/pkg/test"
+)
+
+func inspectImage(image string) ([]byte, error) {
+	args := []string{"inspect", image}
```
Comment (ArangoGutierrez) [inline src/cmd/singularity/inspect_test.go:16]: and here test at least inspect over 2 sections
```
[eduardo@rhel7 singularity]$ singularity inspect --runscript labels.sif

#!/bin/sh

    echo "Running command: $*"
    exec "$@"

[eduardo@rhel7 singularity]$ singularity inspect --deffile labels.sif

mirrorurl: https://www.busybox.net/downloads/binaries/1.26.1-defconfig-multiarch/busybox-x86_64
bootstrap: busybox

%labels
        Version v3.0.0-beta
        Maintainer sylabs

%runscript
    echo "Running command: $*"
    exec "$@"

%post
    echo "Hello from inside the container"

```

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

File: src/cmd/singularity/inspect_test.go (hunk 1 above)
Comment (GodloveD) [reply src/cmd/singularity/inspect_test.go]: well, I think there should already be some labels there by default right?  Like a timestamp of when the container was created.  So I don't know if we need to add any new ones.

**Review by GodloveD — CHANGES_REQUESTED** [review]
So it looks like this is just checking the inspect command with no flags and then checking to see if the function returns error right?  Could it please do the following too?  

1. Run a new test with each of the inspect flags.  Maybe make the flags a string argument to the test function and then run through all of them.

2. Check the text output of the inspect command against some expectation.  I don't think this has to be exhaustive, like an exact match or something.   But the inspect output should contain keywords that can always be found.

File: src/cmd/singularity/inspect_test.go (hunk 1 above)
Comment (bauerm97) [inline src/cmd/singularity/inspect_test.go:26]: Looks like this function doesn't exist

Comment (ArangoGutierrez) [reply src/cmd/singularity/inspect_test.go:26]: it was rm on https://github.com/sylabs/singularity/pull/2128 , to fix #2127
we should all use  `imageBuild()`
```go
	opts := buildOpts{
		force:    true,
		sandbox:  false,
		writable: false,
	}
	if b, err := imageBuild(opts, imagePath, definition); err != nil {
		t.Log(string(b))
		t.Fatalf("unexpected failure: %v", err)
	}
	imageVerify(t, imagePath, true)
```

**Comment by jscook2345** [comment]
I'm going to delete this PR since it was pre-directory reorg. The code is simple enough to add back in later.

**Pull request closed without merging**